lib/v2/apple/radar.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
empty radar rules

Reply:
`/apple` and `/appstore` have same domain name, can I write to different files?